### PR TITLE
CTCP-1075 Split rules into separate pages

### PR DESCRIPTION
**Pull request by crystalmark**

### CODE REVIEW

**Review by PeterBandura — DISMISSED** [review]

File: source/documentation/rules-b.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,668 @@
...
+<b>Technical Description</b>
+
+N/A<br/>
+
+##B1003
+
+<b>Functional Description</b>
+
+In case of 'Road Transport' if the road vehicle (and/or its road trailers) are different to those declared in the initial declaration message, then they will be reported as discrepancies and not as new transport means.
+
```
Comment (fmcgrath) [inline source/documentation/rules-b.html.md.erb:24]: Grammar/style issues - the text in parentheses complicates "they".

Suggestion:
In the case of 'Road Transport', if a road vehicle and/or its trailers are different from those declared in the initial declaration message, they will be reported as discrepancies and not as new transport means.

File: source/documentation/rules-b.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,668 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##B1016
+
+<b>Functional Description</b>
+
+During the Transitional Period, the data item “Customs office at border reference number”, is required for ‘native CC015C’ and ‘native CC013C’ and optional if CC013C or CC015C is the outcome of the upgraded CC013B or CC015B respectively. After the end of the Transitional Period, the data item will be always required.
+
```
Comment (fmcgrath) [inline source/documentation/rules-b.html.md.erb:34]: Grammar/style issues:
- Remove unnecessary comma before "is required".
- Insert "but is" before "optional".
- Insert comma before "respectively".

File: source/documentation/rules-b.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,668 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##B1091
+
+<b>Functional Description</b>
+
+During the Transitional Period (TP), the native CC015C, CC013C and CC170C sent to the National Transit Application should not include the value ‘99’. This value ‘99’ is valid in CL750 during TP, only in case the value ‘99’ is the result of the upgrade by the National Transit Application of a legacy message (e.g. CC015B upgraded into CC015C).
+
```
Comment (fmcgrath) [inline source/documentation/rules-b.html.md.erb:44]: Grammar/style issues:
- Remove unnecessary comma before "only".
- Change "in case" to "if"
- Change "e.g." to "for example".

**Note:** The use of _e.g._, _etc._, _i.e._, and _via_ in CTC documentation is probably going to be a discussion point for the team. However, the [GDS style guide](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style) advises against using Latinate words and abbreviations. The main problem with these words is that they cause problems for non-native English speakers.

File: source/documentation/rules-b.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,668 @@
...
+##B1813
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt; <br/>THEN R0003 and R0006 shall be disabled.
```
Comment (fmcgrath) [inline source/documentation/rules-b.html.md.erb:147]: Consistency issue - This sentence has a full stop but most other sentences here don't see to have full stops. We need to be consistent.

File: source/documentation/rules-c.html.md.erb [hunk 5]
```diff
@@ -0,0 +1,1358 @@
...
+##C0014
+
+<b>Functional Description</b>
+
+IF the last two characters of /*/messageSender are EQUAL to the 3rd and 4rth character of<br/>/*/GuaranteeReference/GRN AND<br/>IF &lt;CD205C-Guarantee.guaranteeType&gt; is in SET {1, 2, 4, 9}<br/>THEN &lt;GUARANTEE REFERENCE-GUARANTOR&gt; = "R"<br/>ELSE &lt;GUARANTEE REFERENCE -GUARANTOR&gt; = "O"
+
+<b>Technical Description</b>
+
+IF the last two characters of /*/messageSender are EQUAL to the 3rd and 4rth character of /*/GuaranteeReference/GRN AND<br/>IF /CD205C/Guarantee/guaranteeType is in SET {1, 2, 4, 9}<br/>THEN /*/GuaranteeReference/Guarantor = "R"<br/>ELSE /*/GuaranteeReference/Guarantor = "O"<br/>
+
```
Comment (fmcgrath) [inline source/documentation/rules-c.html.md.erb:78]: `4th` not `4rth`

File: source/documentation/rules-e.html.md.erb [hunk 6]
```diff
@@ -0,0 +1,218 @@
...
+##E1118
+
+<b>Functional Description</b>
+
+N/A
+
+<b>Technical Description</b>
+
+IF &lt;Decisive Date&gt; is LESS than or EQUAL to &lt;TPendDate&gt;<br/>THEN<br/>/*/Guarantee/GuaranteeReference/accessCode<br/><br/>format shall be set to an4<br/><br/>
+
```
Comment (fmcgrath) [inline source/documentation/rules-e.html.md.erb:138]: Should be `an4` be `an..4`?

File: source/documentation/rules-g.html.md.erb [hunk 7]
```diff
@@ -0,0 +1,1208 @@
...
+---
+#Rules G
+Based on document version 5.15.0-v0.10 and issue date 04/03/2022
+<%= partial 'documentation/partials/rulesintro' %>
+##G0001
+
+<b>Functional Description</b>
+
+If at least one &lt;CONSIGNMENT-HOUSE CONSIGNMENT-ADDITIONAL INFORMATION.Code&gt; is EQUAL to ’30600’ in the declaration, then for the specific &lt;CONSIGNMENT-HOUSE CONSIGNMENT&gt; the Data Group CONSIGNEE shall not be used AND &lt;CONSIGNMENT-CONSIGNEE&gt; shall not be used. For the rest of the repetitions of &lt;CONSIGNMENT-HOUSE CONSIGNMENT&gt; the specific IF statement (“If at least one &lt;CONSIGNMENT-HOUSE CONSIGNMENT-ADDITIONAL INFORMATION.Code&gt; is EQUAL to ’30600’”) shall be re-validated.<br/>During the Transitional Period same approach shall be followed for the &lt;CONSIGNMENT-HOUSE CONSIGNMENT-CONSIGNMENT ITEM-ADDITIONAL INFORMATION.Code&gt;.<br/>
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:14]: Grammar/style issue - Recommend inserting a comma after `For the rest of the repetitions of &lt;CONSIGNMENT-HOUSE CONSIGNMENT&gt;`

File: source/documentation/rules-g.html.md.erb [hunk 8]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0003
+
+<b>Functional Description</b>
+
+This Data Item is omitted if the Data Group &lt;RISK ANALYSIS RESULT&gt; does not refer to a specific Goods Item but is applicable to the whole Consignment.<br/>This Data Item has the value '0' if the Data Group &lt;RISK ANALYSIS RESULT&gt; does not refer to a specific Goods Item but is applicable to all Goods Items included in the Consignment.<br/>This Data Item has a non-zero value if the Data Group &lt;RISK ANALYSIS RESULT&gt; refer to a specific Goods Item.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:34]: Grammar issue - `This Data Item has a non-zero value if the Data Group &lt;RISK ANALYSIS RESULT&gt; refer to a specific Goods Item.` needs to use "refers" (not "refer") because group is singular.

File: source/documentation/rules-g.html.md.erb [hunk 9]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0012
+
+<b>Functional Description</b>
+
+It should include the XPath location of the error. If the XPath string is to be truncated (i.e. if the length of the string is greater than 512 characters long), then the data item should not be used.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:104]: Grammar/style issues -
- See earlier comment in this PR review about use of Latinate words and abbreviations. Suggest changing `i.e.` to `that is,` (note comma).
- Suggest removing "then" because it is a redundant word (the conditionality of the sentence is implied by the comma).

File: source/documentation/rules-g.html.md.erb [hunk 10]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0013
+
+<b>Functional Description</b>
+
+It should be used when the error is an XML schema error concerning invalid values. The reasons for considering an attribute value invalid might be the format and/or a value for a technical code list. For such cases, the data item should contain the value of the invalid value in order to indicate which value was perceived invalid.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:114]: Grammar/style issue - Remove `in order` because it is a redundant phrase.

File: source/documentation/rules-g.html.md.erb [hunk 11]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0015
+
+<b>Functional Description</b>
+
+In case of Incident information received via one or multiple CD180C, all the information received shall be included in the D.G. &lt;CONSIGNMENT-INCIDENT&gt;.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:134]: Grammar/style issues -
- Although `via` is not mentioned in GDS style guide, it is a Latinate word that should be avoided if possible. Suggest rewording to `by means of`.
- The phrase `one or multiple CD180C` is problematic because using multiple forces you to pluralise `CD180C` in some way. Adding an "s" (`CD180Cs`) would be create an ambiguity but perhaps we can add a phrase between `multiple` and `CD180C` to compensate.

File: source/documentation/rules-g.html.md.erb [hunk 12]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0016
+
+<b>Functional Description</b>
+
+&lt;CONSIGNMENT-INCIDENT-TRANSPORT EQUIPMENT.Container identification number&gt; is filled in case of incident with the new Container identification number amending initial declaration or with the existing Container identification number if DI Number of seals is greater than 0.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:144]: Grammar/style issues -
- `in case of incident` does not read well. Can we reword?
- `amending initial declaration` does not read well. Need to insert `an` or `the `before initial.

File: source/documentation/rules-g.html.md.erb [hunk 13]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0017
+
+<b>Functional Description</b>
+
+'State of seals' = 1 in case that Seals are in good state (i.e. present and not damaged)<br/>'State of seals' = 0 in case that Seals are not in good state (i.e. not present or damaged).
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:154]: Grammar/style issue - Change both occurrences of `i.e.` to `that is,` (note comma).

File: source/documentation/rules-g.html.md.erb [hunk 14]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0020
+
+<b>Functional Description</b>
+
+Refers to the mode of transport corresponding to the active means of transport which is expected to be used on exit from or entry into the Safety and Security area.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:174]: Grammar/style issue - Insert comma before `which`.

File: source/documentation/rules-g.html.md.erb [hunk 15]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0021
+
+<b>Functional Description</b>
+
+The value '0' (zero) is a valid number in this Data Item, as per applicable XSD pattern.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:184]: Grammar/style issue - per is another Latinate word. Suggest rewording to `, as defined by the applicable XSD pattern`.

File: source/documentation/rules-g.html.md.erb [hunk 16]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0023
+
+<b>Functional Description</b>
+
+The D.I. will be filled in with new value amending initial declaration in case of incident
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:204]: Grammar/style issues -
- Consider inserting `an` or `the` before `initial`.
- I'm not keen on `in case of incident`. Generally, this sort of phrasing should start a sentence because it is a condition.

Suggestion: If there is an incident, the D.I. will be filled in with the new value to amend the initial declaration.

File: source/documentation/rules-g.html.md.erb [hunk 17]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0030
+
+<b>Functional Description</b>
+
+If a consignment is moving from one Member State to another Member State via a third country which is not in set of CL009 (CountryCodesCommonTransit) then a &lt;CUSTOMS OFFICE OF TRANSIT (DECLARED)&gt; shall be declared and located in the specific Member States. 
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:254]: Multiple problems here (via, which, then).

Suggestion: If a consignment is moving from one Member State to another Member state by means of a third country that is not set in CL009 (CountryCodesCommonTransit), a &lt;CUSTOMS OFFICE OF TRANSIT (DECLARED)&gt; shall be declared and located in the specific Member States.

File: source/documentation/rules-g.html.md.erb [hunk 18]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0026
+
+<b>Functional Description</b>
+
+This D.G./D.I. can be used only  <br/>         - in case of Export followed by transit (maximum one MRN included per one House Consignment) <br/>     AND <br/>         - if the &lt;TRANSIT OPERATION.Additional Declaration Type&gt; is EQUAL to 'A'.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:234]: Again, moving suddenly into an abbreviation - `D.G.`

Will this be clear to novice readers?

File: source/documentation/rules-g.html.md.erb [hunk 19]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0033
+
+<b>Functional Description</b>
+
+The Data Item &lt;AUTHORISATION.Reference number&gt; must be valid in CDMS or in the National Decision Management System.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:274]: Consistency issue - Now, we're using `Data Item` instead of `D.I.`

File: source/documentation/rules-g.html.md.erb [hunk 20]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0034
+
+<b>Functional Description</b>
+
+In case of Export followed by Transit and whenever the internal transit procedure is applied, the Declared Office of Destination needs to be ‘appropriate’, otherwise the initial submission and/or subsequent amendment requests of the transit declaration data as submitted by the Holder of the Transit Procedure to the Office of Departure has to be rejected. This can be validated as follows:<br/>A/ In case the Declared Office of Destination belongs to EU MS (CL010- CountryCodesCommunity), and its Custom Office Reference Number is included in both CL172- CustomsOfficeDestination and CL294-CustomsOfficeExitDeclared, then it is considered ‘appropriate’ (otherwise is considered not ‘appropriate’);<br/>B/ In case the Declared Office of Destination belongs to CTC (CL112- CountryCodesCTC), it is considered by default ‘appropriate’.<br/>When the Declared Office of Destination is considered as not ‘appropriate’, the messages CC013C or CC015C will be responded with CC056C that will report the error code '12- Codelist violation'.<br/>
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:284]: Grammar/style issues - 
- Suggest changing `In case of` to `If` (need to add is after `Export`).
- Suggest changing `In case the` to `If the`.
- Remove comma before `and its`.
- Remove `then`.

File: source/documentation/rules-g.html.md.erb [hunk 21]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0035
+
+<b>Functional Description</b>
+
+The Holders of the Transit Procedure can only request information on their own Guarantee[s].
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:294]: Grammar/style issues - 
- Positioning of `only`.
- Why are square brackets used in `Guarantee(s)`. Parentheses are customary in plain text.

The following rule is not mentioned in GDS style guide but should be followed at all times: **Only goes only before the word it modifies.**

Suggestion: The Holders of the Transit Procedure can request information only about their own Guarantee(s).

File: source/documentation/rules-g.html.md.erb [hunk 22]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0036
+
+<b>Functional Description</b>
+
+COMPETENT CUSTOMS OFFICE AT DEPARTURE which is authorised for Enquiry communicates only with COMPETENT CUSTOMS OFFICE AT DESTINATION which is authorised for Enquiry. COMPETENT CUSTOMS OFFICE AT DEPARTURE which is authorised for Recovery communicates only with COMPETENT CUSTOMS OFFICE AT DESTINATION which is authorised for Recovery.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:304]: These sentences don't read well because of over-use of `which is`. However it might be hard to reword them.

That said, each occurrence of `which is` should be prefixed with a comma.

File: source/documentation/rules-g.html.md.erb [hunk 23]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0041
+
+<b>Functional Description</b>
+
+In a case when National risk is identified, by default, the value ‘X’ shall be included in &lt;RiskAnalysisIdentification.code&gt; when generated by the risk engine. However, if the sender would likely consider such information sent by the Office of Export /Office of Departure as useless for the recipient, then Office of Export /Office of Departure should use the value ‘N’.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:344]: Suggest removing `then`.

File: source/documentation/rules-g.html.md.erb [hunk 24]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0050
+
+<b>Functional Description</b>
+
+The Reference number shall include the ARC number or the fallback eAD reference number when the ‘Type’ of the ‘Additional reference’ is C651 or C658 respectively.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:374]: Is `eAD` correct? Should it not be `EAD`?

Insert a comma before `respectively`.

File: source/documentation/rules-g.html.md.erb [hunk 25]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0061
+
+<b>Functional Description</b>
+
+The information presented in in this D.G. is related to Safety & Security and to the Binding Itinerary. In case of Binding itinerary, the information entered must include the list of codes of the countries between the Office of Departure and the Office of Destination. If more information is available about the countries visited by the means of transport since it's first place of loading until the last place of unloading, it should also be added for Safety & Security purpose only.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:404]: Four issues:
- Repeated word `in`.
- Reword `In case of Binding itinerary` to `For Binding itineraries,` (note comma).
- Change `it's` to `its` (the former is a contraction of `it is` whereas the latter is a possessive).
- The phrase `purpose only` is problematic because it's not good style to put `only` at the end of a sentence. This might require some discussion but I reckon that the last clause of the last sentence should be reworded to `it should also be added but only for Safety & Security purposes`.

File: source/documentation/rules-g.html.md.erb [hunk 26]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0062
+
+<b>Functional Description</b>
+
+The rules R0506 and R0507 are applied on CC015C and CC013C to ensure that the declaration does not include unnecessary and repetitive information. They must be enforced by all NTA. Considering the possibility that one Goods Item is taken out from the declaration during the control, the message CC029C and CD001C may have different content from CC015C (or CC013C or CC170C). Consequently, those rules R0506 and R0507 shall not be strictly enforced on the Common Domain messages. Certainly not by the recipient of the CD message, likely not by the sender of the CD message.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:414]: Two issues - 
`All NTA `does not read well. I would be inclined to avoid using an abbreviation here - or else pluralise abbreviation because of use of `all`.
- The final sentence (starts with Certainly) has the wrong tone and sounds too conversational. Rewording will require discussion.

File: source/documentation/rules-g.html.md.erb [hunk 27]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0064
+
+<b>Functional Description</b>
+
+The message must be sent within thirteen (13) days from the day the “Destination Control Results” C_DES_CON (IE018) message is sent to the Office of Departure
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:424]: Grammar/style issue - Change `the day the` to `the day when the`.

File: source/documentation/rules-g.html.md.erb [hunk 28]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0068
+
+<b>Functional Description</b>
+
+The Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE&gt; will be also used to include the information of EMCS consignment exported from one EU member state into a Non-EU-Member state, in case of Export Followed by Transit (where in messages CC013C or CC015C the &lt;CONSIGNMENT- HOUSE CONSIGNMENT- PREVIOUS DOCUMENT.Type&gt; = ‘N830’ AND &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE. Type&gt; is in SET CL234 (DocumentTypeExcise)).<br/>In this case, the Data Group &lt;GOODS SHIPMENT- GOODS ITEM- PREVIOUS DOCUMENT&gt; of the Export declaration, will be mapped with the Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- ADDITIONAL REFERENCE&gt; of the Transit declaration.   
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:434]: The first sentence is too long and need to be split. Suggest inserting a full stop after `state` and changing `in case of Export` to `If Export is`.

File: source/documentation/rules-g.html.md.erb [hunk 29]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0069
+
+<b>Functional Description</b>
+
+The Data Group &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- SUPPORTING DOCUMENT&gt;, can be also used to include the information related to EMCS consignment (where &lt;CONSIGNMENT- HOUSE CONSIGNMENT- CONSIGNMENT ITEM- SUPPORTING DOCUMENT.Type&gt; is in SET CL234 (DocumentTypeExcise)), transported from one EU member state into another EU member state via a CTC country.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:444]: Change `via` to `by means of`.

File: source/documentation/rules-g.html.md.erb [hunk 30]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0071
+
+<b>Functional Description</b>
+
+In case of Export Followed by Transit (i.e. &lt;CONSIGNMENT-HOUSE CONSIGNMENT- PREVIOUS DOCUMENT. Type&gt; = ‘N830’), all and only the goods items declared in &lt;GOODS SHIPMENT-GOODS ITEM&gt; as defined in the related Export declaration (identified by the MRN) must be included in &lt;CONSIGNMENT-HOUSE CONSIGNMENT- CONSIGNMENT ITEM&gt; Data Group.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:454]: Two issues - 
- Unless `Export Followed by Transit` is phrasing that has to be preserved, suggest changing `In case of Export Followed` to `If Export is Followed`.
- Change `i.e.` to `that is,` (note comma).

File: source/documentation/rules-g.html.md.erb [hunk 31]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0072
+
+<b>Functional Description</b>
+
+In case of Export Followed By Transit (&lt;CONSIGNMENT-HOUSE CONSIGNMENT- PREVIOUS DOCUMENT.Type&gt; = ‘N830’), <br/>-	all the goods items declared in &lt;GOODS SHIPMENT-GOODS ITEM&gt; as defined in the related Export declaration (identified by the MRN) and <br/>-	all the goods items declared in the &lt;CONSIGNMENT-HOUSE CONSIGNMENT- CONSIGNMENT ITEM&gt; defined in the transit declaration <br/>must be listed in the same order (with &lt;GOODS SHIPMENT-GOODS ITEM. Declaration goods item number&gt; = &lt;CONSIGNMENT-HOUSE CONSIGNMENT- CONSIGNMENT ITEM. Goods item number&gt;). Keeping the order of the goods item is required to enable the automatic validation of the matching of the goods in the context of Export followed by Transit.<br/>
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:464]: See previous comment about `In case of Export`.

File: source/documentation/rules-g.html.md.erb [hunk 32]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0073
+
+<b>Functional Description</b>
+
+When the ‘AES communication purpose’ is EQUAL to ‘2’ (= Allocation of the export MRN(s) referenced in the transit declaration), if the previous message CC191C includes one or more excise goods (i.e. &lt;CC191C-AES RESULTS-EXPORT OPERATION-GOODS SHIPMENT&gt; is PRESENT) then it is an external transit procedure (i.e. the &lt;CC190C-TRANSIT OPERATION-EXPORT OPERATION. Transit procedure category&gt; shall be equal to ‘1’= External Transit Procedure) for that (or those) Export MRN(s).
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:474]: Change `i.e.` (two occurrences) to `that js, `(note comma).

File: source/documentation/rules-g.html.md.erb [hunk 33]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0074
+
+<b>Functional Description</b>
+
+The Administrative Reference Code (ARC) or the fallback e-AD reference number shall be recorded in this Data Item.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:484]: See comment above about `eAD`. We need to be consistent.

File: source/documentation/rules-g.html.md.erb [hunk 34]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0102
+
+<b>Functional Description</b>
+
+For each type of authorisation, the authorisation is valid for the whole declaration (i.e. for the different HOUSE CONSIGNMENTS).
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:544]: Change `i.e.` to `that is,` (note comma).

File: source/documentation/rules-g.html.md.erb [hunk 35]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0115
+
+<b>Functional Description</b>
+
+This Data Item is required as per UCC-DA (Annex B) but may be waived for modes of transport other than rail in case the transit movement does not cross the external border of the Union.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:614]: Suggest changing `as per `to `by`.

File: source/documentation/rules-g.html.md.erb [hunk 36]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0125
+
+<b>Functional Description</b>
+
+The D.I. ‘Limit for the enquiry response date’ is equal to ‘Preparation date and time’ of the CD142C message plus either 28 days or 40 days (no other value is acceptable). This means that the date value of the DI is based on the expiration date of the timer “Wait for Enquiry response” set manually by Officer at the Competent Authority of Enquiry at Departure as follows: <br/><br/>’28 days’ after the CD142C is sent: The Holder of the Transit Procedure was contacted and provided insufficient information or in case the Competent Authority of Enquiry suspects fraud. In this case, a reply to the enquiry request is expected within 28 days at the latest.<br/><br/>’40 days’ after the CD142C is sent: The Holder of the Transit Procedure was contacted and provided sufficient information OR the Holder of the Transit Procedure was not contacted since there is sufficient information to initiate enquiry. In this case, a reply to the enquiry request is expected within 40 days at the latest. 
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:674]: Consistency issue - Mixing `D.I.` and `DI`.

File: source/documentation/rules-g.html.md.erb [hunk 37]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A 
+
+##G0170
+
+<b>Functional Description</b>
+
+When the IE is transmitted to two or more Customs Offices that belong to the same National Administration, then only one IE is sent to this National Administration (the Customs Offices can be an Office of Transit or an Office of Exit for Transit or an Office of Destination or a Customs Office of Incident Registration).
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:834]: Can remove `then`.

File: source/documentation/rules-g.html.md.erb [hunk 38]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A
+
+##G0500
+
+<b>Functional Description</b>
+
+The exact content of the CL326 (QualifierOfTheIdentification) is defined nationally, considering -for example -that only in some NAs the value 'T' must only be used in case “House number” and “Postcode” or only “Postcode” define an exact and unique location.
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:1044]: The text considering -for example - is a mess. Suggest splitting sentence by a adding a full stop after `nationally` and starting second sentence with `For example, in only some NAs the...`

File: source/documentation/rules-g.html.md.erb [hunk 39]
```diff
@@ -0,0 +1,1208 @@
...
+<b>Technical Description</b>
+
+N/A<br/>
+
+##G0869
+
+<b>Functional Description</b>
+
+In case of Export Followed by Transit, the content of RISK ANALYSIS IDENTIFICATION received by the Customs Office of Exit should be forwarded via NCTS to the Customs Office of Exit for Transit (or Office of Destination at the border).  <br/>If the Customs Office of Exit for Transit (or Office of Destination at the border) belongs to the same Member State, then the exchange of risk related information is under the responsibility of the National Customs Administration. <br/><br/> 
+
```
Comment (fmcgrath) [inline source/documentation/rules-g.html.md.erb:1144]: See earlier comments about `In case of`.

File: source/documentation/rules-r.html.md.erb [hunk 40]
```diff
@@ -0,0 +1,838 @@
...
+##R0001
+
+<b>Functional Description</b>
+
+Each &lt;CD144C-RESPONSE INFORMATION.Information code&gt; can only be used once per message
+
+<b>Technical Description</b>
+
+Each /CD144C/ResponseInformation/informationCode can only be used once per message
+
```
Comment (fmcgrath) [inline source/documentation/rules-r.html.md.erb:18]: Positioning of `only`.

Change to `can be used only once per message`.

**Review by fmcgrath — CHANGES_REQUESTED** [review]
Have reviewed 8 of the 9 files.

Three overriding problems with the files:
- Inconsistencies with use of certain abbreviations, such as `eAD` and `e-AD` (is EAD correct)?
- Inconsistent references to data items (`Data Item`, `D.I.`, and `DI`) - it's less of an issue for data groups.
- Use of Latinate words and abbreviations (`e.g.`, `etc.`, `i.e.`, `per`, and `via`). This is not good style for internationalisation and GDS style guide recommends avoiding the abbreviations.

File: source/documentation/rules-b.html.md.erb (hunk 1 above)
Comment (crystalmark) [reply source/documentation/rules-b.html.md.erb:24]: This is a direct copy from the DDNTA so we're not able to change the wording.

File: source/documentation/rules-e.html.md.erb (hunk 6 above)
Comment (crystalmark) [reply source/documentation/rules-e.html.md.erb:138]: an4 = 4 characters exactly
an..4 = up to 4 characters.  
I think an4 is right

**Review by fmcgrath — APPROVED** [review]
After discussion with stakeholders, I am approving these changes.
